doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -11,11 +11,11 @@ This tutorial has following sections:
 - [Azure SDK Code Generation Quickstart Tutorial (Data Plane)](#azure-sdk-code-generation-quickstart-tutorial-data-plane)
   - [Prerequisites](#prerequisites)
   - [Setup your repository](#setup-your-repository)
-  - [Create starter package](#create-starter-package)
-    - [Use Cadl as Input](#use-cadl-as-input)
-      - [Create Cadl project](#create-cadl-project)
-      - [Create sdk starter package](#create-sdk-starter-package)
-    - [Use swagger as Input](#use-swagger-as-input)
+  - [Create SDK Package](#create-sdk-package)
+    - [Use Cadl as Input API spec](#use-cadl-as-input-api-spec)
```

Comment:
remove this as a heading, if someone links into this page that is what they are doing already.

Reply:
removed

Reply:
I still see this section in the document

Reply:
`Create SDK Package` section? I removed this.